src/Geosuggest.jsx

```diff
@@ -36,10 +36,22 @@ class Geosuggest extends React.Component {
 
     this.onInputChange = this.onInputChange.bind(this);
     this.onAfterInputChange = this.onAfterInputChange.bind(this);
+    this.onInputFocus = this.onInputFocus.bind(this);
```

Comment:
Why move all class properties with arrow functions to calls methods?

Functions as properties have `this` bound automatically. So no need to bind them explicitly.

Reply:
*prettier* didn’t work with the arrow functions.